source/llassetgen-rendering/WindowQt.cpp

```diff
@@ -127,6 +127,14 @@ bool WindowQt::event(QEvent* event) {
 
 void WindowQt::paintGL() { updateGL(); }
 
+void WindowQt::mousePressEvent(QMouseEvent* /*event*/) {}
+
+void WindowQt::mouseMoveEvent(QMouseEvent* /*event*/) {}
+
+void WindowQt::mouseReleaseEvent(QMouseEvent* /*event*/) {}
+
+void WindowQt::wheelEvent(QWheelEvent* /*event*/) {}
```

Comment:
I think these are all overrides and could be marked as such.

Reply:
Yes. They are now marked as such in the header file.